pkg/ruler/tenant_federation.go

```diff
@@ -20,8 +22,10 @@ type TenantFederationConfig struct {
 	Enabled bool `yaml:"enabled"`
 }
 
+const TenantFederationFlag = "ruler.tenant-federation.enabled"
+
 func (cfg *TenantFederationConfig) RegisterFlags(f *flag.FlagSet) {
-	f.BoolVar(&cfg.Enabled, "ruler.tenant-federation.enabled", false, "Enable running rule groups against multiple tenants. The tenant IDs involved need to be in the rule group's 'source_tenants' field. If this flag is set to 'false' when there are already created federated rule groups, then these rules groups will be skipped during evaluations.")
+	f.BoolVar(&cfg.Enabled, TenantFederationFlag, false, "Enable running rule groups against multiple tenants. The tenant IDs involved need to be in the rule group's 'source_tenants' field. If this flag is set to 'false' when there are already created federated rule groups, then these rules groups will be skipped during evaluations.")
```

Comment:
```suggestion
	f.BoolVar(&cfg.Enabled, TenantFederationFlag, false, "Enable rule groups querying against multiple tenants. The tenant IDs involved need to be in the rule group's 'source_tenants' field. If this flag is set to 'false' when there are already created federated rule groups, then these rules groups will be skipped during evaluations.")
```

Reply:
This is important because we don't write to multiple tenants; we query to N to only write to the one that evaluates it so it's technically a bit more accurate. 

Unsure if this is a breaking change though so it's def a "nit", I believe help text of flags is not part of guarantees.